connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectServer.scala

```diff
@@ -38,8 +39,8 @@ object SparkConnectServer extends Logging {
         SparkConnectService.server.getListenSockets.asScala.foreach { sa =>
           val isa = sa.asInstanceOf[InetSocketAddress]
           logInfo(
-            s"Spark Connect server started at: " +
-              s"${isa.getAddress.getHostAddress}:${isa.getPort}")
+            log"Spark Connect server started at: " +
+              log"${MDC(RPC_ADDRESS, isa.getAddress.getHostAddress)}:${MDC(PORT, isa.getPort)}")
```

Comment:
I'm not sure if calling it `HOST_PORT` would be more appropriate.

Reply:
Either way seems fine. Or, we can consider unifying them.